assignments/F-02-93/js/components/Register.js

```diff
@@ -0,0 +1,89 @@
+import React, {Component} from 'react';
```

Comment:
Move code in `miniproject/pages/register.js`

Reply:
Done